### PR TITLE
Simple foundation for GraphQL

### PR DESCRIPTION
**Pull request by FelixHultman**
- Added dependencies
- Added a GraphQL patch in application.properties just for a initial test

## Summary by CodeRabbit

- **New Features**
	- Integrated GraphQL support, enabling the app to handle GraphQL queries.
	- Added a new GraphQL schema with a simple greeting query.

- **Configuration Changes**
	- Introduced properties for GraphQL schema locations and endpoint path.

- **Dependencies**
	- Added new dependencies for GraphQL functionality and testing utilities.

### CODE REVIEW

**Review by AlbinNils98 — DISMISSED** [review]
Nicely done with the setup, looks good!

You only need to resolve the conflicts with main then this pr is good to go

File: pom.xml [hunk 1]
```diff
@@ -120,11 +120,28 @@
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-oauth2-client</artifactId>
-        </dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-validation</artifactId>
```
Comment (oskber) [inline pom.xml:124]: Looks like this validation dependency is a duplicate?

**Review by oskber — APPROVED** [review]
Setup looks good 👍 Should we add GraphiQL? ("_GraphiQL is a useful visual interface for writing and executing queries, and much more. Enable GraphiQL by adding this config to the application.properties file_ - **spring.graphql.graphiql.enabled=true**.") 
image

**Comment by AlbinNils98** [comment]
As @oskber said, there seems to be a duplication of spring-boot-starter-validation dependency in the pom.xml

**Review by AlbinNils98 — APPROVED** [review]

**Pull request merged**